app/views/inventory_foods/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render partial: 'layouts/flash' %>
+<div class="w-75 mx-auto">
+<div class="d-flex justify-content-between">
+  <p class="fs-1">Inventory: <%= Inventory.find(params[:inventory_id]).name.capitalize %></p>
+  <%= render 'button' %>
+</div>
```

Comment:
It would be better to implement the Model objects in controllers than the views, Remember that we are using the MVC  pattern in Ruby on rails applications.